code/game/gamemodes/bloodbrother/brother.dm

```diff
@@ -0,0 +1,174 @@
+/datum/antagonist/brother
+	name = "Brother"
+	roundend_category = "Brother"
+	job_rank = ROLE_BROTHER
+	var/special_role = ROLE_BROTHER
+	var/give_objectives = TRUE
+	var/datum/team/brother_team/team
+
+/datum/antagonist/brother/create_team(datum/team/brother_team/new_team)
+	if(!new_team)
+		return
+	if(!istype(new_team))
+		stack_trace("Wrong team type passed to [type] initialization")
+	team = new_team
+
+/datum/antagonist/brother/get_team()
+	return team
+
+/datum/antagonist/brother/on_gain()
+	SSticker.mode.brothers += owner
+	owner.objectives += team.objectives
+	owner.special_role = special_role
+	update_brother_icons_added()
+	finalize_brother()
+	return ..()
+
+/datum/antagonist/brother/on_removal()
+	SSticker.mode.brothers -= owner
+	owner.objectives -= team.objectives
+	if(owner.current)
+		to_chat(owner.current,"<span class='userdanger'> You are no longer the [special_role]!</span>")
+	owner.special_role = null
+	update_brother_icons_removed()
+	return ..()
+
+/datum/antagonist/brother/proc/get_brother_names()
+	var/list/brothers = team.members - owner
+	var/brother_text = ""
+	for(var/i = 1 to brothers.len)
+		var/datum/mind/M = brothers[i]
+		brother_text += M.name
+		if(i == brothers.len - 1)
+			brother_text += " and "
+		else if(i != brothers.len)
+			brother_text += " , "
+	return brother_text
+
+/datum/antagonist/brother/proc/give_meeting_area()
+	if(!owner.current || !team || !team.meeting_area)
+		return
+	to_chat(owner.current, "<B>Your designated meeting area:</B> [team.meeting_area]")
+	antag_memory += "<b>Meeting Area</b>: [team.meeting_area]<br>"
+
+/datum/antagonist/brother/proc/get_team_members()
+	
```

Comment:
Unintended?

Reply:
yeah, I was thinking of a way to slip into mind, but best leave that to check antags.